CHANGES.md

```diff
@@ -1,3 +1,7 @@
 # 0.1.0
 
-* First version!
\ No newline at end of file
+* First version!
+
+# 0.2.0
```

Comment:
Nitpick: Usually the newest changes are at the top, so you don't need to scroll all the way down in order to see the latest changes.